renovate.json

```diff
@@ -61,6 +61,20 @@
       "matchBaseBranches": ["stable27", "stable26", "stable25"],
       "enabled": false
     },
+    {
+      "groupName": "tiptap",
+      "matchBaseBranches": ["stable25"],
+      "matchPackagePrefixes": [
+        "@tiptap/",
+        "prosemirror-"
+      ],
+      "enabled": false
+    },
+    {
+      "matchUpdateTypes": ["major"],
```

Comment:
Do we need this block twice?

Reply:
No, copy paste seems harder than expected 🙈 Fixed.

Reply:
Hehe, how about swapping the two blocks, so that the two tiptap blocks are one after each other? Sorry for the nitpicking, feel free to ignore :zany_face:

Reply:
I thought about that but wanted to rather group by stable branches from new to old ;)